Tenable/alsid/tests/alert_certificate_DANG_ACCESS.json

```diff
@@ -0,0 +1,30 @@
+  },
+  "expected": {
+    "message": "\"0\" \"1\" \"ad.domain\" \"ad.domain\" \"C-PKI-DANG-ACCESS\" \"critical\" \"CN=DSCUCNExport,CN=Certificate Templates,CN=Public Key Services,CN=Services,CN=Configuration,DC=ad,DC=domain\" \"1996839\" \"2\" \"R-CERTIF-TEMPLATE-ACL\" \"76485473\"      \n \"DistinguishedName\"=\"CN=DSCUCNExport,CN=Certificate Templates,CN=Public Key Services,CN=Services,CN=Configuration,DC=ad,DC=domain\"                                                                                                             \n \"DangerousAceList\"=\"[{\"Item1\":\"A;;LCRPWPRCWDWO;;;S-1-5-21-1229472208-2678311744-2345022811-345849\",\"Item2\":\"S-1-5-21-1229472208-2678311744-2345022811-345849\",\"Item3\":\"urdom.ad.domain\\\\AC750-DSI-FOO-Espace de                               \n travail-GSW-Adm\",\"Item4\":[{\"Item1\":\"Modify permissions\",\"Item2\":\"\"},{\"Item1\":\"Modify owner\",\"Item2\":\"\"},{\"Item1\":\"Write all properties\",\"Item2\":\"\"}]}]\"",
+    "event": {
+      "kind": "alert"
+    },
+    "action": {
+      "name": "C-PKI-DANG-ACCESS",
+      "outcome": "success",
+      "outcome_reason": "R-CERTIF-TEMPLATE-ACL",
+      "properties": {
+        "ADdevianceID": 1996839,
+        "ADdomainName": "ad.domain",
+        "ADforestName": "ad.domain",
+        "ADobject": "CN=DSCUCNExport,CN=Certificate Templates,CN=Public Key Services,CN=Services,CN=Configuration,DC=ad,DC=domain",
+        "alertID": 1,
+        "alertSeverityLevel": "critical",
+        "eventID": "76485473\"       \"DistinguishedName\"=\"CN=DSCUCNExport,CN=Certificate Templates,CN=Public Key Services,CN=Services,CN=Configuration,DC=ad,DC=domain\"                                                                                                              \"DangerousAceList\"=\"[{\"Item1\":\"A;;LCRPWPRCWDWO;;;S-1-5-21-1229472208-2678311744-2345022811-345849\",\"Item2\":\"S-1-5-21-1229472208-2678311744-2345022811-345849\",\"Item3\":\"urdom.ad.domain\\\\AC750-DSI-FOO-Espace de                                travail-GSW-Adm\",\"Item4\":[{\"Item1\":\"Modify permissions\",\"Item2\":\"\"},{\"Item1\":\"Modify owner\",\"Item2\":\"\"},{\"Item1\":\"Write all properties\",\"Item2\":\"\"}]}]"
```

Comment:
Is the eventID "76485473" or the whole line ?

Reply:
@TOUFIKIzakarya  Good catch thank you!

It should now be fixed